<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMIForm</class>
 <widget class="QWidget" name="TMIForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>108</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>108</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>888</width>
    <height>108</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMain">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="labelSensors">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Interrogation sensors:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxS1">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxS2">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxS3">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxS4">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label1">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Interrogation count:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxIntCount">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>111</width>
           <height>21</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Single</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triple</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label2">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Period interrogation (interval 200ms):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxIntPeriod">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label3">
         <property name="minimumSize">
          <size>
           <width>241</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Cycles count:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxCyCount">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>21</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
